tree.go

```diff
@@ -170,6 +170,7 @@ const (
 	leafType       byte = 2
 	eoAccountType  byte = 3
 	singleSlotType byte = 4
+	skipListType   byte = 8
```

Comment:
this is set in bit order, because I'm hoping to mix encodings in the future.

Reply:
`eoAccountType` has value 3 which isn't respecting that idea. Is it worth changing now? If not, we're ina a half-baked idea.

Reply:
we could change it to 4, no long-term db has been using this.